Add TLS configuration instructions (#33)
aerospike/aerospike-jdbc: docs/tls.md

# Aerospike JDBC TLS Configuration Parameters

The Aerospike JDBC driver can be configured to use SSL through the JDBC connection URL. 

Below are the tls-related configs for aerospike-jdbc (example with value):

> ```tlsEnabled true

Reviewer: New line after ```

Author: have added newline before and after the code block

tlsName TLS_NAME
tlsStoreType (like jks)

> tlsTruststorePassword.   ******* (if required)
> tlsTruststorePath.  ./../XXX.jks (the full path)

Reviewer: ```suggestion
tlsTruststorePassword   ******* (if required)
tlsTruststorePath  ./../XXX.jks (the full path)
```

```
The full list of the valid values can be found at [AerospikeTLSPolicyConfig](https://github.com/aerospike/aerospike-jdbc/blob/main/src/main/java/com/aerospike/jdbc/tls/AerospikeTLSPolicyConfig.java).

> The configuration expects a standard Java truststore. The example to add CA certificate to Java Truststore can be found at [Add CA certificate to Java TrustStore on client nodes](https://docs.aerospike.com/server/operations/configure/network/tls/mtls_java#add-ca-certificate-to-java-truststore-on-client-nodes).

Reviewer: ```suggestion
The configuration expects a standard Java truststore. An example of adding a CA certificate to the Java Truststore can be found at [Add CA certificate to Java TrustStore on client nodes](https://docs.aerospike.com/server/operations/configure/network/tls/mtls_java#add-ca-certificate-to-java-truststore-on-client-nodes).
```

> ```keytool -import -alias tls1 -file /etc/aerospike/ssl/tls1/cert.pem -keystore //usr/lib/jvm/java-8-openjdk-amd64/jre/lib/security/cacerts -storePass changeit

Reviewer: New line after ```

Author: have added new line before and after the code block

```

> Here are some working URL example using the above configs:

Reviewer: ```suggestion
Here are some working URL examples using the above configuration properties:
```

> ```//aerospike server detail

Reviewer: New line after ```

Author: have added new line before and after the code block
